<?xml version="1.0" encoding="UTF-8"?><testrun duration="62738" footerText="Generated by IntelliJ IDEA on 10/1/20, 2:43 PM" name="tests">
    <count name="total" value="23"/>
    <count name="failed" value="2"/>
    <count name="passed" value="21"/>
    <config nameIsGenerated="true" configId="TestNG" name="tests">
        <module name="tafforbilling"/>
        <useClassPathOnly/>
        <extension name="coverage">
            <pattern>
                <option name="PATTERN" value="tests.*"/>
                <option name="ENABLED" value="true"/>
            </pattern>
        </extension>
        <option name="SUITE_NAME" value=""/>
        <option name="PACKAGE_NAME" value="tests"/>
        <option name="MAIN_CLASS_NAME" value=""/>
        <option name="METHOD_NAME" value=""/>
        <option name="GROUP_NAME" value=""/>
        <option name="TEST_OBJECT" value="PACKAGE"/>
        <option name="PARAMETERS" value=""/>
        <option name="OUTPUT_DIRECTORY" value=""/>
        <option name="PROPERTIES_FILE" value=""/>
        <properties/>
        <listeners/>
    </config>
    <root name="Default Suite" location="file://C:\Users\dzamzam\AppData\Local\JetBrains\IdeaIC2020.2\temp-testng-customsuite.xml">
        <output type="stdout">C:\Users\dzamzam\.jdks\openjdk-14.0.2\bin\java.exe -ea -Didea.test.cyclic.buffer.size=1048576 &amp;quot;-javaagent:C:\Users\dzamzam\AppData\Local\JetBrains\IntelliJ IDEA Community Edition 2020.2\lib\idea_rt.jar=61497:C:\Users\dzamzam\AppData\Local\JetBrains\IntelliJ IDEA Community Edition 2020.2\bin&amp;quot; -Dfile.encoding=UTF-8 -classpath &amp;quot;C:\Users\dzamzam\AppData\Local\JetBrains\IntelliJ IDEA Community Edition 2020.2\lib\idea_rt.jar;C:\Users\dzamzam\AppData\Local\JetBrains\IntelliJ IDEA Community Edition 2020.2\plugins\testng\lib\testng-rt.jar;C:\Users\dzamzam\Documents\tafforbilling\target\test-classes;C:\Users\dzamzam\Documents\tafforbilling\target\classes;C:\Users\dzamzam\.m2\repository\org\testng\testng\6.14.3\testng-6.14.3.jar;C:\Users\dzamzam\.m2\repository\com\beust\jcommander\1.72\jcommander-1.72.jar;C:\Users\dzamzam\.m2\repository\org\apache-extras\beanshell\bsh\2.0b6\bsh-2.0b6.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-java\3.12.0\selenium-java-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-api\3.12.0\selenium-api-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-chrome-driver\3.12.0\selenium-chrome-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-edge-driver\3.12.0\selenium-edge-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-firefox-driver\3.12.0\selenium-firefox-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-ie-driver\3.12.0\selenium-ie-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-opera-driver\3.12.0\selenium-opera-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-remote-driver\3.12.0\selenium-remote-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-safari-driver\3.12.0\selenium-safari-driver-3.12.0.jar;C:\Users\dzamzam\.m2\repository\org\seleniumhq\selenium\selenium-support\3.12.0\selenium-support-3.12.0.jar;C:\Users\dzamzam\.m2\repository\net\bytebuddy\byte-buddy\1.8.3\byte-buddy-1.8.3.jar;C:\Users\dzamzam\.m2\repository\org\apache\commons\commons-exec\1.3\commons-exec-1.3.jar;C:\Users\dzamzam\.m2\repository\commons-codec\commons-codec\1.10\commons-codec-1.10.jar;C:\Users\dzamzam\.m2\repository\commons-logging\commons-logging\1.2\commons-logging-1.2.jar;C:\Users\dzamzam\.m2\repository\com\google\code\gson\gson\2.8.2\gson-2.8.2.jar;C:\Users\dzamzam\.m2\repository\com\google\guava\guava\23.6-jre\guava-23.6-jre.jar;C:\Users\dzamzam\.m2\repository\com\google\code\findbugs\jsr305\1.3.9\jsr305-1.3.9.jar;C:\Users\dzamzam\.m2\repository\org\checkerframework\checker-compat-qual\2.0.0\checker-compat-qual-2.0.0.jar;C:\Users\dzamzam\.m2\repository\com\google\errorprone\error_prone_annotations\2.1.3\error_prone_annotations-2.1.3.jar;C:\Users\dzamzam\.m2\repository\com\google\j2objc\j2objc-annotations\1.1\j2objc-annotations-1.1.jar;C:\Users\dzamzam\.m2\repository\org\codehaus\mojo\animal-sniffer-annotations\1.14\animal-sniffer-annotations-1.14.jar;C:\Users\dzamzam\.m2\repository\org\apache\httpcomponents\httpclient\4.5.3\httpclient-4.5.3.jar;C:\Users\dzamzam\.m2\repository\org\apache\httpcomponents\httpcore\4.4.6\httpcore-4.4.6.jar;C:\Users\dzamzam\.m2\repository\com\squareup\okhttp3\okhttp\3.9.1\okhttp-3.9.1.jar;C:\Users\dzamzam\.m2\repository\com\squareup\okio\okio\1.13.0\okio-1.13.0.jar;C:\Users\dzamzam\AppData\Local\JetBrains\IntelliJ IDEA Community Edition 2020.2\plugins\testng\lib\jcommander-1.27.jar&amp;quot; com.intellij.rt.testng.RemoteTestNGStarter -usedefaultlisteners false -socket61496 @w@C:\Users\dzamzam\AppData\Local\Temp\idea_working_dirs_testng.tmp -temp C:\Users\dzamzam\AppData\Local\Temp\idea_testng.tmp
</output>
        <output type="stdout">===============================================
Default Suite
Total tests run: 5, Failures: 2, Skips: 0
===============================================
</output>
        <output type="stdout">Process finished with exit code 0
</output>
    </root>
    <suite duration="62738" locationUrl="file://C:\Users\dzamzam\AppData\Local\JetBrains\IdeaIC2020.2\temp-testng-customsuite.xml" name="tafforbilling" status="failed">
        <suite duration="13912" locationUrl="java:suite://tests.AddMetersTest" name="AddMetersTest" status="failed">
            <test duration="2027" locationUrl="java:test://tests.AddMetersTest/UserCanSyncMeters[0]" name="AddMetersTest.UserCanSyncMeters" status="failed">
                <output type="stdout"/>
                <output type="stderr">java.lang.AssertionError: null
&#13;
	at org.testng.Assert.fail(Assert.java:96)&#13;
	at org.testng.Assert.assertNotEquals(Assert.java:1157)&#13;
	at org.testng.Assert.assertNotEquals(Assert.java:1162)&#13;
	at tests.AddMetersTest.UserCanSyncMeters(AddMetersTest.java:33)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)&#13;
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)&#13;
</output>
            </test>
        </suite>
        <suite duration="13663" locationUrl="java:suite://tests.CreateNewCustomerTest" name="CreateNewCustomerTest" status="passed">
            <test duration="3458" locationUrl="java:test://tests.CreateNewCustomerTest/UserCanCreateNewCustomer[0]" name="CreateNewCustomerTest.UserCanCreateNewCustomer" status="passed">
                <output type="stdout"/>
            </test>
        </suite>
        <suite duration="21160" locationUrl="java:suite://tests.LoginTest" name="LoginTest" status="failed">
            <test duration="397" locationUrl="java:test://tests.LoginTest/UserCanLoginSuccssfully[0]" name="LoginTest.UserCanLoginSuccssfully" status="failed">
                <output type="stdout"/>
                <output type="stderr">java.lang.AssertionError: expected [http://10.0.61.23:9090/#/] but found [http://10.0.61.23:9090/#/login]
</output>
                <output type="stdout">Expected :
</output>
                <output type="stderr">http://10.0.61.23:9090/#/
</output>
                <output type="stdout">Actual   :
</output>
                <diff actual="http://10.0.61.23:9090/#/login" expected="http://10.0.61.23:9090/#/"/>
                <output type="stderr">http://10.0.61.23:9090/#/login
&#13;
	at org.testng.Assert.fail(Assert.java:96)&#13;
	at org.testng.Assert.failNotEquals(Assert.java:776)&#13;
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)&#13;
	at org.testng.Assert.assertEquals(Assert.java:118)&#13;
	at org.testng.Assert.assertEquals(Assert.java:453)&#13;
	at org.testng.Assert.assertEquals(Assert.java:463)&#13;
	at tests.LoginTest.UserCanLoginSuccssfully(LoginTest.java:22)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)&#13;
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)&#13;
</output>
            </test>
            <test duration="492" locationUrl="java:test://tests.LoginTest/UserCanLoginSuccssfullyandsavecredentials[0]" name="LoginTest.UserCanLoginSuccssfullyandsavecredentials" status="passed">
                <output type="stdout"/>
            </test>
        </suite>
        <suite duration="14002" locationUrl="java:suite://tests.MakeNewPaymentTest" name="MakeNewPaymentTest" status="passed">
            <test duration="2775" locationUrl="java:test://tests.MakeNewPaymentTest/UserCanPayforcertainCustomer[0]" name="MakeNewPaymentTest.UserCanPayforcertainCustomer" status="passed">
                <output type="stdout"/>
            </test>
        </suite>
    </suite>
</testrun>
